src/LobbyForm.tsx

```diff
@@ -109,10 +109,10 @@ const hostStyles = StyleSheet.create({
     // minWidth: "230px",
     padding: "16px",
     borderRadius: "8px",
-    background: "#FFFFFF",
+    background: "#1D1E33",
     margin: "auto",
     marginTop: "32px",
-    border: "1px solid #EEEEEE",
+    border: "1px solid #1D1E33",

```

Comment:
same as above, do we need a border? rather no border than a border of the same color as the background